arelle/CntlrCmdLine.py

```diff
@@ -175,8 +175,8 @@ def parseArgs(args):
                       help=_("Specify a disclosure system name and"
                              " select disclosure system validation.  "
                              "Enter --disclosureSystem=help for list of names or help-verbose for list of names and descriptions. "))
-    parser.add_option("--hmrc", action="store_true", dest="validateHMRC",
-                      help=_("Select U.K. HMRC disclosure system validation."))
```

Comment:
Do we want to get rid of this? It seems like we would want to keep this around a for a bit to allow people to switch over to the new UK option without breaking them?

Reply:
`--hmrc` is still available, `--uk` is just another option for the same arg.